src/components/common/form/text-area/notes-summary-list/NotesSummaryList.jsx

```diff
@@ -7,7 +7,6 @@ const NotesSummaryList = () => {
   const [addNote, setAddNote] = useState(false);
 
   const handleClick = () => {
-    event.preventDefault();
```

Comment:
why don't we need this anymore?
Doesn't the default link event behavior cause a refresh or something like that?

Reply:
I know sonarcloud was highlighting it as a code smell, and Joanna thought we didn't need it, but I'm not sure of the exact reason. I can put it back in as it doesn't really affect this PR, and can be looked into at a later date?

Reply:
oh right, looks like you guys confirmed it's all good so no worries. Will resolve.